src/rt/sections_elf_shared.d

```diff
@@ -753,6 +753,7 @@ void scanSegments(in ref dl_phdr_info info, DSO* pdso) nothrow @nogc
 
         case PT_TLS: // TLS segment
             assert(!pdso._tlsSize); // is unique per DSO
+            version (CRuntime_UClibc) {} else
```

Comment:
To follow up on @joakim-noah's suggestion, I would prefer the following style here.
```
version (CRuntime_UClibc)
{
    // comment explaining why.
} 
else
    pdso._tlsMod = info.dlpi_tls_modid;
```
As it makes the version block stand out in the code, rather than hidden away.

Reply:
Changed as suggested. Thanks!